code/datums/keybinding/xeno.dm

```diff
@@ -169,19 +175,14 @@
 	full_name = "Lay Egg"
 	description = ""
 	keybind_signal = COMSIG_XENOABILITY_LAY_EGG
+	hotkey_keys = list("\\")
```

Comment:
Stick to letters please. No \`, no \\. These are not the same across keyboards and might be difficult (for example, for me ` is altgr + 7, and \ is altgr + q. Not ideal)

Reply:
Sure thing, that's been changed now.